### ISSUE TITLE
Coral Dev Board Mini cannot booting up

### ISSUE THREAD

**Issue by IfaizulHasan** [google-coral/edgetpu#577]
### Description

Hello, I have a problem with my coral dev board.

Before this, I had used this board smoothly but somehow today when I tried to turn it on, the orange indicator lit up. I read the datasheet stated that there was a possibility of boot failure.

The chronology was on yesterday I installed opencv-python-headless for about 12 hours with HDMI attached. Then when I turn off the board and tried to turn it on again, it didn't appear anything on the screen and only the orange indicator lit up as I said earlier.

I tried to do flushing as stated on the Coral website but was still unsuccessful, Is my board broken?

<details><summary>Click to expand!</summary> 
 
 ### Issue Type

Support

### Coral Device

Dev Board Mini

### Relevant Log Output

_No response_</details>

**Comment by hjonnala** [comment]
Hello @IzuruSakamaki have you tried to re flash the board using [fastboot](https://coral.ai/docs/dev-board-mini/reflash/#force-boot-into-fastboot-mode) method? 

can you please provide the following details:
- host operating system used to connect with dev board mini
- board refalshing logs
- serial console logs

Thanks!

**Comment by ooshyun** [comment]
I got a same issue in my coral dev board mini. I am currently using MacOS M1 Pro, and before this, I connected this device and ran ssh service. Finishing my work, I unplug this device. After those situations, the host OS cannot recognize this device when turned on. This device lit orange only, not change to green.

I tried fast boot, but it did not recognize it when running bash enable_lk_fastboot.sh. Is there any way to recover this or solve it?

**Comment by Pablorzayastuy** [comment]
@ooshyun did you ever find a solution to this issue? We are having the same problem as you.

**Comment by ooshyun** [comment]
@Pablorzayastuy No, I want to debug in software and hardware side, and I tried fastboot and reflash way, but I failed. If success, then please share the way!

**Comment by Pablorzayastuy** [comment]
@ooshyun I managed to fix it by force booting into fastboot using a jumper wire (https://coral.ai/docs/dev-board-mini/reflash/#force-boot-into-fastboot-mode). After that the device was recognized through USB serial connection, and the light turned green again.
